src/helix-ui/styles/core/hx-icon.less

```diff
@@ -6,4 +6,9 @@ hx-icon {
   height: 1em;
   width: 1em;
   vertical-align: -2px;
+
+  svg {
```

Comment:
This was originally in shadow dom as `::slotted(svg)` which is the same as `hx-icon svg` in the light dom, so it was pull out of the shadow dom.